xml/en/docs/windows.xml

```diff
@@ -8,10 +8,17 @@
 <article name="nginx for Windows"
          link="/en/docs/windows.html"
          lang="en"
-         rev="5">
+         rev="6">
 
 <section>
 
+<para>
+<note>
+Currently nginx for Windows is available in a beta version
```

Comment:
Shouldn't the 'beta' mention in the net paragraph be adjusted, too? There is no reason to mention beta more than once. It's good to lift it to the top of the article.